tests/test-rest-nav-menus-controller.php

```diff
@@ -80,32 +296,100 @@ public function test_get_item_links() {
 		$this->assertEquals( $location_url, $links['https://api.w.org/menu-location'][0]['href'] );
 	}
 
+	/**
+	 *
+	 */
 	public function test_get_items_no_permission() {
 		wp_set_current_user( 0 );
-		$request = new WP_REST_Request( 'GET', '/wp/v2/menus' );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/menus' );
 		$response = rest_get_server()->dispatch( $request );
 		$this->assertErrorResponse( 'rest_cannot_view', $response, 401 );
 	}
 
+	/**
+	 *
+	 */
 	public function test_get_item_no_permission() {
 		wp_set_current_user( 0 );
-		$request = new WP_REST_Request( 'GET', '/wp/v2/menus/' . $this->menu_id  );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/menus/' . $this->menu_id );
 		$response = rest_get_server()->dispatch( $request );
 		$this->assertErrorResponse( 'rest_cannot_view', $response, 401 );
 	}
 
+	/**
+	 *
+	 */
 	public function test_get_items_wrong_permission() {
 		wp_set_current_user( self::$subscriber_id );
-		$request = new WP_REST_Request( 'GET', '/wp/v2/menus' );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/menus' );
 		$response = rest_get_server()->dispatch( $request );
 		$this->assertErrorResponse( 'rest_cannot_view', $response, 403 );
 	}
 
+	/**
+	 *
+	 */
 	public function test_get_item_wrong_permission() {
 		wp_set_current_user( self::$subscriber_id );
-		$request = new WP_REST_Request( 'GET', '/wp/v2/menus/' . $this->menu_id  );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/menus/' . $this->menu_id );
 		$response = rest_get_server()->dispatch( $request );
 		$this->assertErrorResponse( 'rest_cannot_view', $response, 403 );
 	}
 
+	/**
+	 * @param $response
```

Comment:
And types here as well and in the other methods.

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/41/commits/565a06fff290cddae77b2167a2d05b457e85e1d6